src/contexts/Maci.tsx

```diff
@@ -232,8 +232,6 @@ export const MaciProvider: React.FC<MaciProviderProps> = ({ children }: MaciProv
       return;
     }
 
-    setIsLoading(true);
```

Comment:
Why did you remove loading logic?

Reply:
because it caused something strange: the `isLoading` changed, affect the exported value, the `useAppState` changed, the `Info` keeps on refreshing.

Reply:
Deleting loading logic is not a fix, it can be something related to render or hook/context dependencies.

Reply:
So I think the problem is 
https://github.com/privacy-scaling-explorations/maci-rpgf/blob/c280402b2e2ce1513e8aab928a058939d503f13e/src/contexts/Maci.tsx#L264
--> this shows that the `useEffect` hook depends on `votingEndsAt`
however,
https://github.com/privacy-scaling-explorations/maci-rpgf/blob/c280402b2e2ce1513e8aab928a058939d503f13e/src/contexts/Maci.tsx#L87
--> `votingEndsAt` depends on `pollData`
and actually `pollData` is set here
https://github.com/privacy-scaling-explorations/maci-rpgf/blob/c280402b2e2ce1513e8aab928a058939d503f13e/src/contexts/Maci.tsx#L244
which is inside the same `useEffect` hook

So it causes a loop that causing `isLoading` become `true` and `false` so many times, which affect the `useAppState` hook, that also affect the `Info` component.

Reply:
I added some condition to avoid `setPollData` every time, but not sure if it's a good way. Need some recommendation.

Reply:
No, it's a good way to prevent it.